### ISSUE TITLE
x11 running two window creation/deletion panics

### ISSUE THREAD

**Issue by thiolliere** [tomaka/winit#421]
```rust
extern crate winit;

fn main() {
    for _ in 0..2 {
        let mut events_loop = winit::EventsLoop::new();

        let _window = winit::WindowBuilder::new()
            .with_title("A fantastic window!")
            .build(&events_loop)
            .unwrap();

        events_loop.poll_events(|event| {
        });
    }
}
```

it panics on unwrap None at https://github.com/tomaka/winit/blob/master/src/platform/linux/x11/mod.rs#L360 
and then if I fix that it panics on https://github.com/tomaka/winit/blob/master/src/platform/linux/x11/mod.rs#L719

It seems that FocusOut and ConfigureNotify events are sent after window deletion.
A solution would be to return when `windows.get_mut(..)` returns None on those events. Or maybe on all events.

**Comment by francesca64** [comment]
This appears to be a duplicate of #414, which means this should be resolved by #416.

(Also, this isn't related to your issue, but as general usage advice, my understanding is that you should only create one `EventsLoop` and use it for all windows.)

**Comment by thiolliere** [comment]
#416 doesn't solve the issue panic happens at https://github.com/francesca64/winit/blob/x11-drop-window/src/platform/linux/x11/mod.rs#L375

(i agree for one eventsloop)

**Comment by francesca64** [comment]
I've updated the code such that all cases should be covered now. Does everything work fine now?

**Comment by thiolliere** [comment]
yes thanks !

**Comment by francesca64** [comment]
Fixed via #416
